.github/workflows/ci-cd.yml

```diff
@@ -374,6 +374,80 @@ jobs:
       if: ${{ matrix.os == 'ubuntu' }}
       uses: aio-libs/prepare-coverage@v24.9.2
 
+
+  test_debug_mode:
+    name: Test in debug mode
+    strategy:
+      matrix:
+        pyver:
+        - 3.13t
+        - 3.13
+        - 3.12
+        - 3.11
+        - "3.10"
+        - 3.9
+      fail-fast: false
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+
+    steps:
+    - name: Checkout project
+      uses: actions/checkout@v4
```

Comment:
Could you change this to checkout from sdist instead of Git? It'll mimic what both pip and downstreams do. And this is what we really want not to crash.

Reply:
The issue was not about 'cannot build from sdist' but 'cannot build and run tests when debug mode is on'.

I don't think that the installation from sdist makes any difference here.

Reply:
Downstreams tend to additionally package debug symbols.

Reply:
What symbols? Could you please elaborate how it does related?